keyboards/model_f_labs/f62/wcass/info.json

```diff
@@ -0,0 +1,629 @@
+{
+    "bootloader": "atmel-dfu",
+    "build": {
+        "lto": true
+    },
+    "features": {
+        "backlight": false,
+        "bootmagic": false,
+        "command": true,
+        "console": false,
+        "extrakey": true,
+        "haptic": true,	
+        "mousekey": true,
+        "nkro": true,
+        "raw": true,
+        "rgblight": false
+    },
+    "indicators": {
+        "caps_lock": "B5",
+        "num_lock": "B4"
+    },
+    "processor": "atmega32u2",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x6262",
+        "vid": "0x1209"
+    },
+     "community_layouts": [
+        "60_ansi_tsangan",
+        "60_tsangan_hhkb",
```

Comment:
Just a heads up, `60_tsangan_hhkb` will be deprecated in favour of `60_ansi_tsangan_split_bs_rshift` as of #24186.

Reply:
I assume what is needed depends on which PR is the first that is accepted.  I will wait and see what gets accepted.

Reply:
The RP for 60_tsangan_hhkb has been accepted. I will update the F62 source.

Reply:
60_tsangan_hhkb has been replace with 60_ansi_tsangan_split_bs_rshift